README.md

```diff
@@ -27,6 +23,10 @@ emphasize minimalism even more. Some of them are as follows:
 Some of these might be added later and [PR's are always
 welcomed](https://github.com/ejmg/zerm/pulls).
 
+## differences from ejmg port
+
+- math typesetting support with KaTeX.
```

Comment:
This is a merge request *to* the ejmg port, so all of this "fork" / "differences" talk shouldn't be pulled in.

Reply:
My bad; now it should be okay: I added a paragraph which explains the new feature and leaved the other parts of the README as before.